views/jobs/job_card.html

```
@@ -1,4 +1,4 @@
-<div class="w-full shadow-sm rounded-xl p-3 mb-3 backdrop-blur-sm bg-white/30 hover:bg-white/60 dark:bg-gray-900/30 dark:hover:bg-gray-900/60 transition-colors border border-gray-200/50 dark:border-gray-700/50">
+<div class="w-full">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)